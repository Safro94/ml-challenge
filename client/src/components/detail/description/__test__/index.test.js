import React from 'react';

import { render } from '@testing-library/react';

import Description from '../';

describe('Description', () => {
	it('should render OK', () => {
		//Arrange
		const description = '<h2>description</h2>';
		//Act
		const { container } = render(<Description description={description} />);

		//Assert
		expect(container).toMatchSnapshot();
	});
});
