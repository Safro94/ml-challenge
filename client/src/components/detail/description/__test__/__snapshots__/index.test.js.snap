// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Description should render OK 1`] = `
<div>
  <div
    class="container"
  >
    <h2
      class="title"
    >
      Descripci√≥n del producto
    </h2>
    <p
      class="description"
    >
      <h2>
        description
      </h2>
    </p>
  </div>
</div>
`;
